server/.env_template

```diff
@@ -0,0 +1,2 @@
+SUGGESTIC_API_KEY=
```

Comment:
Looks like Development isn't up to date with Main, hence these changes being here. If we fix this elsewhere, happy to revert these changes here.